docs/contributor-guide/developer-guide/getting-started.md

```diff
@@ -141,28 +141,32 @@ If you'd like to run a specific test or set of tests, and/or save sbt startup ti
     ```
     testOnly services.question.types.QuestionDefinitionTest
     ```
+    
+### Running typescript unit tests
+
+To run the unit tests in [app/assets/javascripts](https://github.com/civiform/civiform/tree/main/server/app/assets/javascripts), run the following:
+
+```
+bin/run-ts-tests
+```
+
+If you'd like to run a specific test or set of tests, run the following:
+
+```
+bin/run-ts-tests file1.test.ts file2.test.ts
+```
 
 ### Running browser tests
 
 To run the browser tests (includes all the [Playwright](https://playwright.dev) tests in [`civiform/browser-test/src/`](https://github.com/civiform/civiform/tree/main/browser-test/src), there are three steps:
 
-1.  Build the Docker image for running the playwright tests. This only needs to be done once:
-
-    ```
-    bin/build-browser-tests
-    ```
-2.  Bring up the local test environment using the AWS emulator. This step can be done in a separate terminal window while the above step is still building. Leave this running while you are working for faster browser test runs:
+1.  Bring up the local test environment using the AWS emulator. This step can be done in a separate terminal window while the above step is still building. Leave this running while you are working for faster browser test runs:
```

Comment:
The second sentence is incorrect as it references the step you removed.

I assume that step is in fact not actually needed anymore?